pandas/io/parsers.py

```diff
@@ -60,11 +60,18 @@
 
 Parameters
 ----------
-filepath_or_buffer : str, pathlib.Path, py._path.local.LocalPath or any \
-object with a read() method (such as a file handle or StringIO)
-    The string could be a URL. Valid URL schemes include http, ftp, s3, and
-    file. For file URLs, a host is expected. For instance, a local file could
-    be file://localhost/path/to/table.csv
+filepath_or_buffer : str, path object, or file object
+    Any valid string path is acceptable. The string could be a URL. Valid
+    URL schemes include http, ftp, s3, and file. For file URLs, a host is
+    expected. A local file could be: file://localhost/path/to/table.csv.
+
+    If you want to pass in a path object, pandas accepts the following:
+
+    * pathlib.Path
+    * py._path.local.LocalPath
```

Comment:
Is this supposed to be private? Not familiar with this but if that's the case don't feel like its worth mention here

Reply:
The alias is `py.path.local`, so it is in fact public.  However, the alias resolves to that "private class", so no need to provide indirection for something like this.

Reply:
Then I would use the public alias? or do they use it like that as well in their documentation?

Since it are only two, I would maybe put it in a normal paragraph instead of the bullet points (now it seems to get more weight than the other paragraphs)